Tools/Pipeline/Common/AssemblyAttributes.cs

```diff
@@ -81,6 +81,14 @@ public static string AssemblyCompany
 				return ((AssemblyCompanyAttribute)attributes[0]).Company;
 			}
 		}
+
+        public static string[] Developers
+        {
+            get
+            {
+                return new string[] { "cra0zy (Harry)", "dellis1972 (Dean Ellis)", "dneelyep (Daniel Neel)", "emarcotte (Eugene Marcotte)", "JamesLupiani (James Lupiani)", "jamesford42 (James Ford)", "KonajuGames (Steve 'Sly' Williams)", "mgarstenauer (Martin Garstenauer)", "nightst4r (Mark)", "nkast (Nikos Kastellanos)", "polsys (Petri Laarne)", "RayBatts (Ray Batts)", "Shqrdx (Yuri Roubinsky)", "thiagoabreu (Thiago 'Jedi' Abreu)", "tomspilman (Tom Spilman)" };
```

Comment:
Or we can redirect people to the http://www.monogame.net/about or contributors page:

 https://github.com/mono/MonoGame/graphs/contributors

Of which @cra0zy is already the 13th largest contributor to MonoGame in the few short months he has been helping with the Pipeline tool and Linux... thanks @cra0zy!

Actually I need to refresh the MonoGame About page anyway...  I will be sure to link the GitHub contributors page to it.